code/__HELPERS/lists.dm

```diff
@@ -675,9 +675,9 @@
 
 #define UNSETEMPTY(L) if(L && !L.len) L = null
 #define LAZYREMOVE(L, I) if(L) { L -= I; if(!L.len) { L = null; } }
-#define LAZYADD(L, I) if(!L) { L = list(); } L += I;
```

Comment:
iirc we have it like this on purpose, because BYOND has a limit to how deep defines can go, so we should minimize use of defines in defines, at least for commonly used ones like this, that can already been found in other defines